awx/main/tasks.py

```diff
@@ -507,7 +507,7 @@ def awx_periodic_scheduler():
 
         invalid_license = False
         try:
-            access_registry[Job](None).check_license()
+            access_registry[Job](None).check_license(quiet=True)
```

Comment:
there's probably a way to prove that you couldn't get the job created to this point with a invalid license and so we may not need this particular check, but... meh